sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -191,4 +191,20 @@ public partial class ProtocolClientOptions : Azure.Core.ClientOptions
     {
         public ProtocolClientOptions() { }
     }
+    public sealed partial class ServiceError
```

Comment:
@tg-msft FYI

Reply:
AzureError was another naming option I considered but we don't have a ton of Azure-prefixed types in core.